From 2e7e480d624469429fa33953d0a4684cba235121 Mon Sep 17 00:00:00 2001
From: Christian Jaeger <ch@christianjaeger.ch>
Date: Wed, 5 Feb 2025 22:32:00 +0100
Subject: .chj-home/init 000: [PATCH 182/191] .chj-home/init: follow BROWSER env var protocol

Which I now know is a PATH like string of binary names to try,
separated by ":". No splitting on space is done.
---
 .chj-home/init | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.chj-home/init b/.chj-home/init
--- a/.chj-home/init
+++ b/.chj-home/init
@@ -75,7 +75,7 @@ if [ -e .bash_profile_local ]; then
 fi
 cat <<'EOF' > .bash_profile_local
 export EDITOR=e
-export BROWSER="firefox --new-window"
+export BROWSER="firefox--new-window"
 export EMAIL=$(cat ~/.chj-home_email)
 export LANG=en_GB.UTF-8
 EOF
